<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>266</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Puerto</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="puerto1">
     <property name="maximum">
      <number>999999999</number>
     </property>
     <property name="value">
      <number>8000</number>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Mensaje</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="enviar1">
     <property name="text">
      <string>Enviar</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Puerto</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="puerto2">
     <property name="maximum">
      <number>999999999</number>
     </property>
     <property name="value">
      <number>8001</number>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Mensaje</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLineEdit" name="msj2">
     <property name="text">
      <string>&quot;$C1STS,IC,STA1,,,,,,,e00*12\r\n&quot;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="enviar2">
     <property name="text">
      <string>Enviar</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Puerto</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="puerto3">
     <property name="maximum">
      <number>999999999</number>
     </property>
     <property name="value">
      <number>8002</number>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Mensaje</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLineEdit" name="msj3">
     <property name="text">
      <string>&quot;$C1STS,IC,STA2,,,,,,,e00*12\r\n&quot;</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QPushButton" name="enviar3">
     <property name="text">
      <string>Enviar</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Puerto</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="puerto4">
     <property name="maximum">
      <number>999999999</number>
     </property>
     <property name="value">
      <number>8003</number>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Mensaje</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLineEdit" name="msj4">
     <property name="text">
      <string>&quot;$C1STS,IC,STA3,,,,,,,e00*12\r\n&quot;</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QPushButton" name="enviar4">
     <property name="text">
      <string>Enviar</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLineEdit" name="msj1">
     <property name="text">
      <string>&quot;$I1STS,I/O,DDU,,,,,,,e00*CS\r\n&quot;</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
